src/ducks/balance/Balance.jsx

```diff
@@ -188,8 +196,13 @@ class Balance extends PureComponent {
     this.stopRealtime(TRIGGER_DOCTYPE, this.fetchTriggers)
   }
 
-  fetchAccounts() {
-    const client = this.props.client
+  async fetchAccounts() {
+    const { client } = this.props
+    if (__TARGET__ === 'mobile') {
+      const pouchManager = client.links[0].pouches
+      pouchManager.stopReplicationLoop()
+      await pouchManager.startReplicationLoop()
```

Comment:
We repeat ourselves, we should extract these lines in a method.

Reply:
Done in https://github.com/cozy/cozy-banks/pull/1189/commits/76248a3889af03b5d73e95a40b91ebb5621aab41